examples/cifarMetaformer.py

```diff
@@ -31,6 +31,7 @@ def __init__(
         num_classes=10,
         dim=384,
         attention="scaled_dot_product",
+        feedforward="MLP",
```

Comment:
I'm not sure about the defaults here, how to show that you can use these to repro "Visual Attention" for instance ? Should we show different presets  ?